.circleci/soltest_all.sh

```diff
@@ -54,12 +43,7 @@ echo "Running steps $RUN_STEPS..."
 
 STEP=1
 
-# Run SMTChecker tests separately, as the heaviest expected run.
-[[ " $RUN_STEPS " == *" $STEP "* ]] && EVM=istanbul OPTIMIZE=1 ABI_ENCODER_V1=1 BOOST_TEST_ARGS="-t smtCheckerTests/*" "${REPODIR}/.circleci/soltest.sh"
-STEP=$((STEP + 1))
-
-# Run without SMTChecker tests.
-[[ " $RUN_STEPS " == *" $STEP "* ]] && EVM=istanbul OPTIMIZE=1 ABI_ENCODER_V1=1 BOOST_TEST_ARGS="-t !smtCheckerTests" "${REPODIR}/.circleci/soltest.sh"
+[[ " $RUN_STEPS " == *" $STEP "* ]] && EVM=istanbul OPTIMIZE=1 ABI_ENCODER_V1=1 "${REPODIR}/.circleci/soltest.sh"
```

Comment:
I think this run is not necessary anymore, right?

Reply:
I think you and @cameel agreed on that, I was just not sure I got the argument. There is still the `ABI_ENCODER_V1` variable that is not set for the `istanbul` run in the loop. But if you think it's OK to remove this run, I will do it.

Reply:
Well, I think we agreed initially but then I noticed the variable and said that it matters after all :)

I think this should stay, at least until we drop encoder v1. Normally, when a test does not explicitly state the encoder version it runs with v2. This variable makes `soltest` choose encoder v1 instead. So by removing it, we lose a lot of coverage with encoder v1.

Reply:
OK, so, the special step for encoder v1 is needed, but I think we can run it without the SMTChecker, because SMTChecker is not affected by the ABI encoder, and SMTChecker tests will run for `istanbul` in the loop.